packages/babel-cli/package.json

```diff
@@ -28,7 +28,7 @@
     "commander": "^4.0.1",
     "convert-source-map": "^2.0.0",
     "fs-readdir-recursive": "^1.1.0",
-    "glob": "^7.2.0",
+    "glob": "condition:BABEL_8_BREAKING ? ^10.3.12 : ^7.2.0",
```

Comment:
```suggestion
    "glob": "condition:BABEL_8_BREAKING ? ^10.3.12 : ^7.2.0 (esm:sync)",
```

this should fix CI

Reply:
Thanks. I just figured out the same change from the conditionParser.ts.